tests/conftest.py

```diff
@@ -82,3 +100,14 @@ def profile_test_client():
     with TestClient(app) as client:
         yield client
     app.dependency_overrides = {}
+
+
+@pytest.fixture
+def caplog(_caplog):  # noqa: F811
```

Comment:
Could the argument name be changed to something which isn't almost the same as the function name?